### ISSUE TITLE
Do you have any pre-trained model file?

### ISSUE THREAD

**Issue by lhq1208** [kuangliu/pytorch-cifar#46]
Have you uploaded the pre-trained model file?So I can save a lot of time training them.Thanks very much!

**Comment by felixkreuk** [comment]
+1,

For some reason, I follow the LR instructions but can reproduce the same accuracy for ResNet18. It would be great of someone could upload a pre-trained model.

Thanks!

**Comment by manishravula** [comment]
+1

It would be very useful if we all could start-off with a pre-trained model file. Maybe people who already have trained using this repo's code could pitch in? 

I've just started training, and would love to share my models once they reach a good accuracy.
